ci-operator/step-registry/ovn/conf/baremetal-dualstack/ovn-conf-baremetal-dualstack-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
this file path and name show `baremetal-dualstack`, should be better update to `vsphere`

Reply:
yes, renamed to vsphere because of the specific DHCPv6 range.